app/src/main/java/com/kamron/pogoiv/CPRange.java

```diff
@@ -0,0 +1,15 @@
+package com.kamron.pogoiv;
+
+/**
+ * Created by Johan on 2016-08-18.
+ */
+public class CPRange {
+    public CPRange(int high, int low, double level) {
+        this.high = high;
+        this.low = low;
+        this.level=level;
+    }
+
+    public int high, low;
```

Comment:
Normally ,fields are listed before methods and constructors.

One declaration per line is recommended since it encourages commenting.
```java
public int high;
public int low;
```